cmd/utils/flags.go

```diff
@@ -1719,7 +1719,7 @@ func SetEthConfig(ctx *cli.Context, stack *node.Node, cfg *ethconfig.Config) {
 		cfg.TransactionHistory = ctx.Uint64(TransactionHistoryFlag.Name)
 	} else if ctx.IsSet(TxLookupLimitFlag.Name) {
 		log.Warn("The flag --txlookuplimit is deprecated and will be removed, please use --history.transactions")
-		cfg.TransactionHistory = ctx.Uint64(TransactionHistoryFlag.Name)
```

Comment:
The original intention was to keep the `txlookuplimit` flag but ignore it with a warning log printed.
It can lead user to switch to the new flag.

With this change, it still uses the value from `txlookuplimit` flag, and print a warning log: hey, please switch to new flag.

But I do get the point, @holiman @fjl any thoughts about it?

Reply:
I think the PR is correct. We need to support the old flag still.